ms-campus-kid-school/src/controllers/careers_controller.go

```diff
@@ -0,0 +1,75 @@
+package controllers
+
+import (
+	"ms_campus_kid_school/src/connection"
+	"ms_campus_kid_school/src/utils"
+)
+
+func CreateCareer(career utils.Career) error {
+	bd, err := connection.GetDB()
```

Comment:
All variables should be in English
```suggestion
	db, err := connection.GetDB()
```